actionmailer/test/test_helper_test.rb

```diff
@@ -383,7 +389,7 @@ def test_assert_enqueued_email_with_with_no_block_with_args
     end
   end
 
-  def test_assert_enqueued_email_with_with_parameterized_args
+  def test_assert_enqueued_email_with_params
```

Comment:
We might want to add a second "with" on these tests to be consistent with the rest of the tests. I think they're trying to say "test `assert_enqueued_email_with` with parameterized args."

Reply:
Good point - I gave it a shot.